test/tests/functional/pbs_multiple_execjob_launch_hook.py

```diff
@@ -92,6 +94,18 @@ def test_multi_execjob_hook(self):
                                      ATTR_o: '/tmp/'})
         job.set_sleep_time(1)
         jid = self.server.submit(job)
-        self.mom.log_match(
+        retval = self.mom.log_match(
             "Modified progname value got updated from launch1",
             max_attempts=3, interval=3)
+        self.assertTrue(retval)
+
+        hook1 = ("execjob_hook1", "execjob_hook2")
```

Comment:
Just curious, is there a reason you made this a tuple instead of a list?

Reply:
It should have been just a list. Updated.